src/packaging/docker-services/nodetool-ssl/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM cassandra-ssl
```

Comment:
I'm unsure if `nodetool` and `nodetool-ssl` can be combined, but give it a quick go?

Reply:
Done.